src/components/FlowRunGraph.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <div class="flow-run-graph">
+    <template v-if="load">
+      <RunGraph
+        v-model:viewport="viewport"
+        v-model:selected="selected"
+        v-model:fullscreen="fullscreen"
+        :config="config"
+        class="flow-run-graph__graph p-background"
+      />
+    </template>
+    <template v-else>
+      <FlowRunGraphConfirmation @confirm="confirm" />
+    </template>
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { NodeSelection, RunGraph, RunGraphConfig, ViewportDateRange } from '@prefecthq/graphs'
+  import { useColorTheme } from '@prefecthq/prefect-design'
+  import { computed, ref } from 'vue'
+  import FlowRunGraphConfirmation from '@/components/FlowRunGraphConfirmation.vue'
+  import { useTaskRunsCount } from '@/compositions/useTaskRunsCount'
+  import { useWorkspaceApi } from '@/compositions/useWorkspaceApi'
+  import { FlowRun } from '@/models/FlowRun'
+  import { ServerStateType } from '@/models/StateType'
+
+  const OPT_IN_THRESHOLD = 2000
```

Comment:
So this const is used to determine whether the graph loads automatically? And if the number of task runs exceeds the threshold we provide a disclaimer?

Reply:
This might be worth a note or a more descriptive variable name

Reply:
Updated